src/sagemaker/model_card/model_card.py

```diff
@@ -208,6 +211,257 @@ def search_model_associated_model_cards(model_id: str):
         return cls(**kwargs)
 
 
+class ModelPackageCreator(_DefaultToRequestDict, _DefaultFromDict):
+    """Information about the user who created the model package"""
+
+    def __init__(self, user_profile_name: Optional[str] = None):
+        """Initialize the ModelPackageCreator object.
+
+        Args:
+            user_profile_name (str, optional): The name of the user's profile.
+        """ # noqa E501 # pylint: disable=line-too-long
+        self.user_profile_name = user_profile_name
+
+
+class SourceAlgorithm(_DefaultToRequestDict, _DefaultFromDict):
+    """Source algorithm object"""
+
+    def __init__(self, algorithm_name: str, model_data_url: Optional[str] = None):
+        """Initialize source algorithm object
+
+        Args:
+            algorithm_name (str): The ARN of an algorithm resource that was used to create the model package.
+            model_data_url (str, optional): The Amazon S3 path where the model artifacts, which result from model training, are stored.
```

Comment:
missing `(default: None)`  => `model_data_url (str, optional): The Amazon S3 path where the model artifacts, which result from model training, are stored. (default: None)`